### PR TITLE
amam/payment_method

### PR DESCRIPTION
**Pull request by mustofa-binary**
# Description

add payment method page

Changes:

- add new payment method page
- use accordion
- new table component
- add payment method in the footer

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/payment-methods/index.js [hunk 1]
```diff
@@ -92,81 +92,100 @@ const PaymentMethods = () => (
...
+                            // eslint-disable-next-line no-console
+                            console.log({ ...payment_data }, pd, idx)
+
+                            return (
+                                <AccordionItem
+                                    key={idx}
+                                    style={{
+                                        padding: '2.2rem 4.8rem',
+                                        position: 'relative',
+                                        paddingBottom: pd.note ? '5rem' : '2.2rem',
```
Comment (sara-fs) [inline src/pages/payment-methods/index.js]: padding_bottom or paddingButtom?

File: src/pages/payment-methods/index.js [hunk 2]
```diff
@@ -92,81 +92,100 @@ const PaymentMethods = () => (
...
+                                    }}
+                                    parent_style={{
+                                        marginBottom: '4rem',
```
Comment (sara-fs) [inline src/pages/payment-methods/index.js]: Same for here.

File: src/components/elements/table.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+`
+const TC = styled.div`
+    display: flex;
+    max-width: ${props => props.max_width};
+    grid-area: ${props => props.grid_area};
+    flex-direction: column;
+`
+const TR = styled.div`
+    padding: 0.8rem 0;
+    background-color: ${props => (props.isTitle === 'true' ? 'var(--color-grey-8)' : 'unset')};
```
[the remark this answers is not in the document]
Comment (mustofa-binary) [reply src/components/elements/table.js]: here is the link for naming conventions https://github.com/binary-com/deriv-app/blob/dev/docs/JavaScript/README.md#naming-conventions

File: src/pages/payment-methods/index.js (hunk 1 above)
Comment (mustofa-binary) [reply src/pages/payment-methods/index.js]: this is css in JS, it only accept name like that 🙏 , can take a look at this https://cssinjs.org/?v=v10.0.4 for reference

File: src/pages/payment-methods/index.js (hunk 2 above)
Comment (mustofa-binary) [reply src/pages/payment-methods/index.js]: same for here 👍

**Pull request merged**